### PR TITLE
[9.0][MIG] portal_claim: Renamed to website_crm_claim

### PR DESCRIPTION
**Pull request by cubells**
Features are merged in crm_claim module

### CODE REVIEW

File: openerp/addons/base/migrations/9.0.1.3/pre-migration.py [hunk 1]
```diff
@@ -49,6 +49,7 @@ def cleanup_modules(cr):
             ('contacts', 'mail'),
             ('marketing_crm', 'crm'),
             ('email_template', 'mail'),  # mail_template class
+            ('portal_claim', 'crm_claim'),
```
Comment (pedrobaeza) [inline openerp/addons/base/migrations/9.0.1.3/pre-migration.py]: Looking at the code at 8.0 and 9.0, it's better to just rename (in apriori.py) `portal_claim` to `website_crm_claim`

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

File: addons/crm_claim/migrations/9.0.1.0/pre-migration.py [hunk 2]
```diff
@@ -17,7 +17,15 @@
...
+    openupgrade.rename_columns(env.cr, column_renames)
+    openupgrade.rename_tables(env.cr, table_renames)
+    # Portal_claim is merged here and It is needed rename or delete ir.rule
+    if openupgrade.is_module_installed(env.cr, 'website_crm_claim'):
+        openupgrade.rename_xmlids(env.cr, [
+            ('portal_claim.portal_claim_rule',
+             'website_crm_claim.portal_claim_rule'),
+        ])
+    else:
+        env.ref('portal_claim.portal_claim_rule').unlink()
```
Comment (MiquelRForgeFlow) [inline addons/crm_claim/migrations/9.0.1.0/pre-migration.py]: Delete this `else` because, in this case, there would be nothing to unlink

**Review by MiquelRForgeFlow — CHANGES_REQUESTED** [review]

**Comment by MiquelRForgeFlow** [comment]
And please, update documentation

File: addons/crm_claim/migrations/9.0.1.0/pre-migration.py [hunk 3]
```diff
@@ -17,7 +17,13 @@
...
 
-@openupgrade.migrate()
-def migrate(cr, version):
-    openupgrade.rename_columns(cr, column_renames)
-    openupgrade.rename_tables(cr, table_renames)
+@openupgrade.migrate(use_env=True)
+def migrate(env, version):
+    openupgrade.rename_columns(env.cr, column_renames)
+    openupgrade.rename_tables(env.cr, table_renames)
+    # Portal_claim is merged here and It is needed rename xml-id
```
Comment (pedrobaeza) [inline addons/crm_claim/migrations/9.0.1.0/pre-migration.py]: This is not needed anymore making the renaming in the other place. Please restore it as it was.

File: addons/crm_claim/migrations/9.0.1.0/pre-migration.py [hunk 4]
```diff
@@ -17,7 +17,12 @@
...
 
-@openupgrade.migrate()
-def migrate(cr, version):
-    openupgrade.rename_columns(cr, column_renames)
-    openupgrade.rename_tables(cr, table_renames)
+@openupgrade.migrate(use_env=True)
+def migrate(env, version):
+    openupgrade.rename_columns(env.cr, column_renames)
+    openupgrade.rename_tables(env.cr, table_renames)
+    if openupgrade.is_module_installed(env.cr, 'website_crm_claim'):
```
Comment (pedrobaeza) [inline addons/crm_claim/migrations/9.0.1.0/pre-migration.py]: I'm talking about leaving this as it was before this PR. You don't need to do this with the renaming of the module.

Comment (MiquelRForgeFlow) [reply addons/crm_claim/migrations/9.0.1.0/pre-migration.py]: True :+1:

File: addons/crm_claim/migrations/9.0.1.0/pre-migration.py [hunk 5]
```diff
@@ -17,7 +17,7 @@
 ]
 
 
-@openupgrade.migrate()
-def migrate(cr, version):
-    openupgrade.rename_columns(cr, column_renames)
-    openupgrade.rename_tables(cr, table_renames)
+@openupgrade.migrate(use_env=True)
```
Comment (pedrobaeza) [inline addons/crm_claim/migrations/9.0.1.0/pre-migration.py]: You're still modifying this script. Please restore it as it was for making the diff smaller.

**Comment by cubells** [comment]
You are right @pedrobaeza 

Sorry.

**Comment by pedrobaeza** [comment]
Agh, I miss to say to you about the documentation... Please put `Done - Renamed to website_crm_claim`

**Comment by pedrobaeza** [comment]
And you have to fill also `website_crm_claim` line with `Done - Renamed from portal_claim`

**Review by MiquelRForgeFlow — APPROVED** [review]

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**